heartbeat/monitors/active/icmp/stdloop.go

```diff
@@ -392,7 +392,7 @@ func createListener(name, network string) *icmp.PacketConn {
 	// XXX: need to check for conn == nil, as 'err != nil' seems always to be
 	//      true, even if error value itself is `nil`. Checking for conn suppresses
 	//      misleading log message.
-	if conn == nil && err != nil {
+	ia conn == nil && err != nil {
```

Comment:
Wondering if there's a typo? There should be no such keyword so far.

Reply:
Yup, this was accidentally included. I've just removed it :)